.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Ansible Integration Test
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+  merge_group:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+permissions:
+  contents: read
+jobs:
+  ansible_test:
+    runs-on: ubuntu-latest
+    container: fedora:latest
+    services:
+      testvcenter:
+        image: vmware/vcsim
```

Comment:
I think it should be a separate vcsim env for each test (as a prepare_lab role )